easybuild/easyconfigs/b/Bazel/Bazel-0.4.4.eb

```diff
@@ -0,0 +1,30 @@
+# @author: Robert Schmidt (OHRI)
+# @author: Guilherme Peretti-Pezzi (CSCS) 
+
+easyblock = "CmdCp"
+
+name = 'Bazel'
+version = '0.4.4'
+
+homepage = 'http://bazel.io/'
+description = """Bazel is a build tool that builds code quickly and reliably. 
+It is used to build the majority of Google's software."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+dependencies = [
+    ('Java', '1.8.0_121'),
+]
+
+sources = ['%(namelower)s-%(version)s-dist.zip']
+source_urls = ['https://github.com/bazelbuild/bazel/releases/download/%(version)s']
+cmds_map = [('.*', "JAVA_VERSION=1.8 CC=gcc CXX=g++ ./compile.sh")]
```

Comment:
please leave a blank line above `cmds_map`, and try to avoid hardcoding the Java version; you can use `%(javashortver)s` instead (cfr. http://easybuild.readthedocs.io/en/latest/version-specific/easyconfig_templates.html#template-names-values-for-short-software-versions)